### PR TITLE
Akshay office hours page

### PR DESCRIPTION
**Pull request by akshayrangasaid**
> Pull Request Template
>
> Use this template to quickly create a well written pull request. Delete all quotes before creating the pull request.

## Description

Created a dedicated office hours page in the github repo.

Fixes # (issue)
> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

No need to test as this is part of documentation

### CODE REVIEW

**Comment by hetunandu** [comment]
Needs more emojis 😅

File: office_hours.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<h2 align="center">
+  Appsmith Office Hours
+  </h2>
+  <p align = "center">
+  <a href = "https://app.appsmith.com/applications/602b8aef12ba0d29d3ec151c/pages/602b8aef12ba0d29d3ec151e"> <b>Sign Up For Office Hours</b> </a>
+  </p>
+  
+-----------------
+  
+Appsmith office hours are conducted by the appsmith team every week for 1 hour at 10am CET on Tuesday and 10am PST on Fridays. The office hours are a dedicated time slot in which our team will be available to help our users real time. The office hours also give us an opportunity to interact with the community, understand your concerns and communicate our thinking to you.
```
Comment (Nikhil-Nandagopal) [inline office_hours.md:10]: First line should probable say what office hours are and then talk about the schedule?

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
@akshayrangasaid left just one feedback

File: office_hours.md (hunk 1 above)
Comment (akshayrangasaid) [reply office_hours.md:10]: Makes sense

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Pull request merged**